crates/bevy_ui/src/ui_node.rs

```diff
@@ -391,3 +391,48 @@ pub struct CalculatedClip {
     /// The rect of the clip
     pub clip: bevy_sprite::Rect,
 }
+
+/// The border radius of the node
+/// 
+/// This doesn't require a [`Border`] component
```

Comment:
Why not? This doesn't seem useful without a [`Border`] component.

Reply:
You can change the the "border" radius of any element. If it's a green square with no border, it will be a green square with rounded corners. Maybe this should be name `CornersRadius` instead or something similar?

Reply:
Ah, `CornerRadius` is much clearer! Swap to that, and then explain this in the doc comment :)